va/va_backend.h

```diff
@@ -486,8 +486,15 @@ struct VADriverVTable
             void               *descriptor      /* out */
         );
 
+        VAStatus
+        (*vaQueryOutputValue)(
+            VADriverContextP    ctx,
+            VAContextID         context,          /* in */
```

Comment:
could also add the surfaceID '**VASurfaceID         render_target**' in this interface? it maybe more flexiable for usage.